Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -440,7 +440,7 @@ class BrowserViewController: UIViewController,
 
         didStartAtHome = tabManager.startAtHomeCheck()
         if didStartAtHome {
-            guard presentedViewController != nil else { return }
+            guard !dismissFakespotIfNeeded(), presentedViewController != nil else { return }
```

Comment:
This `guard` should be removed as it can result in not loading tabs that were opened in background because it is exiting the method. I suggest to add the checks from the guard to the `if` statement surrounding it. Also it would be good to add tests for this.